### PR TITLE
:memo: add more details about installing victoire from scratch

### PR DESCRIPTION
**Pull request by lenybernard**
## Type
documentation

## Purpose
Explane how to setup victoire/victoire in a new project

## BC Break
NO

### CODE REVIEW

File: Bundle/CoreBundle/Resources/doc/SETUP.md [hunk 1]
```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
 ```
 
 - Enable the `localizeddate` filter:
+# Fresh install including base config and dependencies
+
+## symfony 2.8
+
+```sh
+mkdir -p ~/workspace/clientName/projectName && cd ~/workspace/clientName/projectName
+symfony new . 2.8
```
Comment (alexislefebvre) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: I prefer to use Composer personally, because I already have Composer and I don't want to use Symfony's own installer: http://symfony.com/doc/current/setup.html#creating-symfony-applications-with-composer

What do you think?

Or we can add a link to http://symfony.com/doc/current/setup.html and let users follow the Symfony official guide.

File: Bundle/CoreBundle/Resources/doc/SETUP.md [hunk 2]
```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
...
+git add .
+git commit -m ":tada: fresh install"
+git push -u origin develop
+```
 
+## Victoire and its dependencies
+
+### Composer
+```
+composer require victoire/victoire \
```
Comment (alexislefebvre) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: It's easier to copy and paste if it's on one line:

`composer require victoire/victoire friendsofsymfony/user-bundle:~2.0@dev sensio/distribution-bundle:^2.3 doctrine/orm --update-with-dependencies`

File: Bundle/CoreBundle/Resources/doc/SETUP.md [hunk 3]
```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
...
+
+`app/config/config.yml`
 ```yml
-twig.extension.intl:
-    class: Twig_Extensions_Extension_Intl
-    tags:
-        - { name: twig.extension }
+framework:
+   ...
+   translator:      { fallbacks: ["%locale%"] }
```
Comment (alexislefebvre) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: Please remove unnecessary spaces:

    :      {

File: Bundle/CoreBundle/Resources/doc/SETUP.md [hunk 4]
```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
...
+    fos_js_base_url: https://troopers.agency
+    victoire_redis_path: redis://127.0.0.1:6379
+    #if locale_pattern is domain
+    locale_pattern_table:
+        agency.troopers.dev: fr
+        troopers.agency: fr
 ```
 
-- Start Redis server:
+Think to update the `parameters.yml` with correct values.
```
Comment (alexislefebvre) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: Remove *Think to*? It implies that it has to be done in the future, but it has to be done immediately.

File: Bundle/CoreBundle/Resources/doc/SETUP.md [hunk 5]
```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
...
-Use the following information to login and start to create your website:
+```sh
+php bin victoire:viewReference:generate -e=dev
+```
 
-|Login|Password|
-|-----|--------|
-|`anakin@victoire.io`|test|
+#### Do you prefer the fixtures way ?
+There is some fixtures in `vendor/victoire/victoire/Tests/Functionnal/src/Acme/AppBundle/DataFixtures/Seeds/ORM/LoadFixtureData.php`. These are used in the victoire behat tests so you can't use them directly from your project but feel free to start from it by copying/pasting it into your own project.
```
Comment (alexislefebvre) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: There *are* some fixtures?

File: Bundle/CoreBundle/Resources/doc/SETUP.md [hunk 6]
```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
...
+	(13, 7, 'Homepage', 'home', 'en'),
+	(14, 7, 'Page d\'accueil', 'accueil', 'fr');
+```
 
-- Log in:
+### Generate view references
 
-Use the following information to login and start to create your website:
+```sh
+php bin victoire:viewReference:generate -e=dev
```
Comment (alexislefebvre) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: `bin/console`?

File: Bundle/CoreBundle/Resources/doc/SETUP.md (hunk 3 above)
Comment (paulandrieux) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: This seems not related to victoire

File: Bundle/CoreBundle/Resources/doc/SETUP.md [hunk 7]
```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
...
+`app/config/config.yml`
 ```yml
-twig.extension.intl:
-    class: Twig_Extensions_Extension_Intl
-    tags:
-        - { name: twig.extension }
+framework:
+   ...
+   translator:      { fallbacks: ["%locale%"] }
+   serializer: { enable_annotations: true }
```
Comment (paulandrieux) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: I think the serializer is only used with the SearchWidget. If so, this line belongs to the SearchWidget doc

File: Bundle/CoreBundle/Resources/doc/SETUP.md [hunk 8]
```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
...
 
+`app/config/config.yml`
 ```yml
 imports:
     - { resource: @VictoireCoreBundle/Resources/config/config.yml }
-```
 
-- Install Redis:
+assetic:
+    use_controller: false
```
Comment (paulandrieux) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: This choice belongs to the developer, victoire does not require it

File: Bundle/CoreBundle/Resources/doc/SETUP.md [hunk 9]
```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
...
+    bundles: ~
+    filters:
+        cssrewrite: ~
+        less: ~ #Victoire needs less under v3.0.0
+
+fos_user:
+    db_driver: orm
+    firewall_name: main
+    user_class: Victoire\UserBundle\Entity\User
+    from_email:
```
Comment (paulandrieux) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: Do not setup this if not required

File: Bundle/CoreBundle/Resources/doc/SETUP.md [hunk 10]
```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
...
+            logging: false
+
+stof_doctrine_extensions:
+    default_locale: en_US
+    orm:
+        default:
+            tree: true
+            sluggable: true
+            timestampable: true
+            softdeleteable: true
```
Comment (paulandrieux) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: softdeletable is not required for victoire

File: Bundle/CoreBundle/Resources/doc/SETUP.md [hunk 11]
```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
...
+
+victoire_core:
+    user_class: "Victoire\\UserBundle\\Entity\\User"
+    business_entity_debug: true
+    layouts:
+        defaultLayout: "Default layout"
+
+victoire_i18n:
```
Comment (paulandrieux) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: This config should not be required

File: Bundle/CoreBundle/Resources/doc/SETUP.md [hunk 12]
```diff
@@ -158,60 +84,284 @@ stof_doctrine_extensions:
...
-   clients:
-       victoire:
-           type: predis
-           alias: victoire
-           dsn: redis://localhost
+`app/config/parameters.yml.dist`
+```
+parameters:
+    ...
+    router.request_context.host: https://troopers.agency
```
Comment (paulandrieux) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: There should be no mentions to troopers

Comment (paulandrieux) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: Why the router parameters are required ?

File: Bundle/CoreBundle/Resources/doc/SETUP.md (hunk 9 above)
Comment (lenybernard) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: this is required since some days ago

File: Bundle/CoreBundle/Resources/doc/SETUP.md (hunk 12 above)
Comment (lenybernard) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: i don't remember, for the js router ?

Comment (paulandrieux) [inline Bundle/CoreBundle/Resources/doc/SETUP.md]: maybe, I suspect absolute url generation in emails (when there is not request context), so not related to victoire
